src/app/shared/components/categories/categories.component.ts

```diff
@@ -0,0 +1,23 @@
+import { Component, OnInit } from '@angular/core';
+
+// Import Post service & model.
+import { PostService } from '../../services/post/post.service';
+import { Post } from '../../services/models/post';
+
+// Import Observable.
```

Comment:
If you're using github copilot make sure to remove the comments after using it. And you should at least one time do it by yourself :)

Reply:
This is just how I structure my imports so that I can see them more easily.